pkg/config/config_template.yaml

```diff
@@ -655,6 +668,12 @@ api_key:
     #
     # loader: core
 
+    ## @param min_collection_interval - number - optional - default: 15
+    ## This changes the collection interval of the check. For more information, see:
+    ## https://docs.datadoghq.com/developers/write_agent_check/#collection-interval
+    #
+    # min_collection_interval: 15
```

Comment:
It looks like this got documented twice, and `discovery_allowed_failures` not documented?

Reply:
@djmitche `discovery_allowed_failures` is already documented in `snmp_listener.discovery_allowed_failures`, but we didn't declared it as known config, an oversight from [previous PR](https://github.com/DataDog/datadog-agent/pull/8227).

There are two different `min_collection_interval`:
- `snmp_listener.min_collection_interval`
- `snmp_listener.configs[].min_collection_interval`

Reply:
Ah, got it.  And apologies for missing the difference in indentation :)

Reply:
Perhaps we can add a note to the `snmp_listener.configs[].min_collection_interval` description clarifying that this config is specific to each instance in `snmp_listener.configs[]`.

Reply:
@fanny-jiang Sure, thanks for the review.

Added some precisions here: https://github.com/DataDog/datadog-agent/pull/8880/commits/d354acdf4abab9a17f0c1a943fc1f9e90f693620